src/config/layers.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * A style's layers property lists all the layers available in that style.
+ * The type of layer is specified by the "type" property, and must be one
+ * of background, fill, line, symbol, raster, circle, fill-extrusion,
+ * heatmap, hillshade.
+ *
+ * Except for layers of the background type, each layer needs to refer to
+ * a source. Layers take the data that they get from a source, optionally
+ * filter features, and then define how those features are styled.
+ *
+ * See: https://docs.mapbox.com/mapbox-gl-js/style-spec/layers/
+ */
+
+export default [
```

Comment:
Do you think we should add a description attribute for each layer? That could be useful for showing a short blurb. Not sure. 

Would it make sense to also figure out a way to describe the legend here?

Reply:
Good idea, I'll keep it in mind!